app/templates/login.hbs

```diff
@@ -124,37 +126,41 @@
     onClose=closeModal
     translucentOverlay=true
   }}
-    <h2>Forgot password</h2>
+    <h2 class="modal-header">Forgot password</h2>
     {{#let completeForgotPassword as |task|}}
       <form onsubmit={{prevent-default (perform task)}}>
         <legend>
           You should receive an email with a confirmation code. Enter it here and
           choose a new password
         </legend>
         <div>
-          <label>code:</label>
+          {{!-- <label>code:</label> --}}
           <input
             type="text"
             required
-            oninput={{action (mut modalContext.code) value="target.value"}}>
+            oninput={{action (mut modalContext.code) value="target.value"}}
+            placeholder="Enter the 6-digit confirmation code">
         </div>
         <div>
-          <label>password:</label>
+          {{!-- <label>password:</label> --}}
           <input
             type="password"
             minlength="8"
             required
-            oninput={{action (mut modalContext.password) value="target.value"}}>
+            oninput={{action (mut modalContext.password) value="target.value"}}
+            placeholder="Enter your new password">
         </div>
-        <input
+        <div>
+          <input
+          type="submit"
```

Comment:
The `<input>`'s attributes should be indented by two spaces

Reply:
Fixed it. Thanks